test/functional/tests/volumes/test_many_lvms_on_single_core.py

```diff
@@ -0,0 +1,114 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import datetime
+
+import pytest
+from storage_devices.lvm import Lvm, LvmConfiguration
+
+from api.cas import casadm
+from api.cas.init_config import InitConfig
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+from test_tools import fs_utils
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import IoEngine, ReadWrite, VerifyMethod
+from test_utils.size import Size, Unit
+
+opencas_conf_path = "/etc/opencas/opencas.conf"
+lv_count = 16
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeSet([DiskType.optane, DiskType.nand]))
+def test_many_lvms_on_single_core():
+    """
+        title: Test for LVM creation on CAS: 1 cache, 1 core, 16 lvms.
+        description: |
+          Validation of LVM support, LVMs created on CAS devices using configuration:
+          1 cache, 1 core, 16 lvms (1 PV, 1 VG, 16 LV).
```

Comment:
What do you think about adding to title and  description an information that LVMs are zreated on CAS device? (like in the test name)